frontend/app/services/user-service.server.ts

```diff
@@ -0,0 +1,55 @@
+import moize from 'moize';
+
+import { getAuditService } from '~/services/audit-service.server';
+import { getInstrumentationService } from '~/services/instrumentation-service.server';
+import { getEnv } from '~/utils/env.server';
+import { getLogger } from '~/utils/logging.server';
+
+const log = getLogger('user-service.server');
+
+export const getUserService = moize(createUserService, { onCacheAdd: () => log.info('Creating new user service') });
+
+function createUserService() {
+  const { CDCP_API_BASE_URI } = getEnv();
+
+  async function createUserWithAttributesAndEmail(email: string, userId: string) {
```

Comment:
It would make sense to rename this function since `Attributes` aren't being passed, but rather an email:

```suggestion
  async function createUserWithEmail(email: string, userId: string) {
```


Or to even be more concise IMO:
```suggestion
  async function createUser(email: string, userId: string) {
```

Reply:
done as suggested